test/message/stdin_messages.out

```diff
@@ -9,7 +9,7 @@ SyntaxError: Strict mode code may not include a with statement
     at Object.<anonymous> ([stdin]-wrapper:*:*)
     at Module._compile (internal/modules/cjs/loader.js:*:*)
     at evalScript (internal/bootstrap/node.js:*:*)
-    at Socket.<anonymous> (internal/bootstrap/node.js:*:*)
+    at Socket.process.stdin.on (internal/bootstrap/node.js:*:*)
```

Comment:
Fun fact: somehow this change and the arrow function change help V8 infer the name of the process stdin callbacks